locales/ja_JP/pcsx2_Iconized.po

```diff
@@ -1,36 +1,26 @@
-# SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR PCSX2 Dev Team
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
You could've update the copyright header while you were at it, I think some files had them so you can check them for a reference.

Reply:
ahahaha. I will.